### PR TITLE
Trimming Verbiage for PSR Revival

### PR DESCRIPTION
**Pull request by ashnazg**

### CODE REVIEW

**Comment by ashnazg** [comment]
@jaapio @kguest @samdark

**Review by samdark — APPROVED** [review]

File: proposed/phpdoc-tags.md [hunk 1]
```diff
@@ -693,16 +690,10 @@ function or method. When provided, it MUST contain a "Type"
...
-delimiting.
-
-It is RECOMMENDED to use this tag with every function and method. An exception to
-this recommendation, as defined by the Coding Standard of any individual project,
-MAY be:
-
-   **functions and methods without a `return` value**: the @return tag MAY be
-   omitted here, in which case an interpreter MUST interpret this as if
-   `@return void` is provided.
+It is RECOMMENDED to use this tag with every function and method.
```
Comment (jaapio) [inline proposed/phpdoc-tags.md]: Should we not rephrase this to:

It is RECOMMENDED to use this tag with every function or method where PHP native types are insufficent.

**Review by jaapio — APPROVED** [review]
